<?xml version="1.0"?>
<launch>

    <!-- roomba_dynamixel_controller -->
    <node pkg="roomba_dynamixel_controller" type="roomba_dynamixel_controller" name="roomab_dynamixel_controller" >
        <!-- remap -->
        <remap from="joint_in"  to="dynamixel_workbench/joint_states" />
        <remap from="angle_in"  to="angle" />
        <remap from="joint_out" to="dynamixel_workbench/joint_trajectory" />

        <!-- params -->
        <param name="IS_TF" value="$(arg is_tf)" />
        <rosparam command="load" file="$(arg config_path)/param/roomba_dynamixel_controller.yaml" />
    </node>

</launch>